extras/dynflow-sidekiq.rb

```diff
@@ -8,9 +8,6 @@
 rails_env_file = File.expand_path('./config/environment.rb', rails_root)
 require rails_env_file
 
-redis_url = SETTINGS.dig(:dynflow, :redis_url)
```

Comment:
is this neded? all settings afair first read from env variables.. am i mistaken @timogoebel ?

Reply:
The settings have to be registered, see #6826.